demo/simple.html

```diff
@@ -429,6 +429,66 @@
 			<div class="screenshots screenshot-d2l-dynamic">
 				<d2l-table id="d2l-dynamic"></d2l-table>
 			</div>
+			<div class="screenshots screenshot-small-sticky">
+				<d2l-table-wrapper sticky-headers><table class="d2l-table">
+					<thead>
+					<tr>
+						<th sticky><span> </span></th>
+						<th><span> </span></th>
+						<th><span> </span></th>
+					</tr>
+					</thead>
+					<tfoot>
+					<tr>
+						<th sticky><span> </span></th>
+						<th><span> </span></th>
+						<th><span> </span></th>
+					</tr>
+					</tfoot>
+					<tbody>
+					<tr>
+						<th sticky><span> </span></th>
+						<td><span> </span></td>
+						<td><span> </span></td>
+					</tr>
+					<tr>
+						<th sticky><span> </span></th>
+						<td><span> </span></td>
+						<td><span> </span></td>
+					</tr>
+					</tbody>
+				</table></d2l-table-wrapper>
+			</div>
+			<div class="screenshots screenshot-d2l-small-sticky">
+				<d2l-table sticky-headers>
```

Comment:
How do you set sticky-headers for d2l-* elements?

Reply:
We had a conversation offline with @dlockhart about that and we're going to open an issue to add it but since d2l-* isn't part of our requirements and would require significant time, we'll leave it as an open issue in case someone finds need of it.